bin/inx-chronicle/src/inx/worker.rs

```diff
@@ -48,19 +48,14 @@ impl Actor for InxWorker {
     type State = InxClient<Channel>;
     type Error = InxWorkerError;
 
-    async fn init(&mut self, cx: &mut ActorContext<Self>) -> Result<Self::State, Self::Error> {
+    async fn init(&mut self, _cx: &mut ActorContext<Self>) -> Result<Self::State, Self::Error> {
         log::info!("Connecting to INX at bind address `{}`.", self.config.connect_url);
-        let mut inx_client = Inx::connect(&self.config).await?;
+        let inx_client = Inx::connect(&self.config).await?;
 
         log::info!("Connected to INX.");
-        let node_status = inx_client.read_node_status(NoParams {}).await?.into_inner();
 
-        if !node_status.is_healthy {
-            log::warn!("Node is unhealthy.");
-        }
-        log::info!("Node is at ledger index `{}`.", node_status.ledger_index);
-
-        cx.spawn_child(InxListener::new(inx_client.clone())).await;
+        // TODO: turn this back on!
```

Comment:
do dis

Reply:
hey we can always sync when it's part of the past :trollface: